CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [7.1.0] 2022-07-01
+
+[Release link on GitHub](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0)
```

Comment:
I'm not sure about the "Release link on GitHub" line. Shouldn't the link be directly on the version? Something like:
```suggestion
## [[7.1.0](https://github.com/iExecBlockchainComputing/iexec-result-proxy/releases/tag/v7.1.0)] 2022-07-01
```

Reply:
Note: just seen @jeremyjams comment. It looks way more readable for me without the `Release link on GitHub` line.

Reply:
done in 6dc7b0c